packages/analyticsdx-template-lint/src/schemas/readiness-schema.json

```diff
@@ -10,7 +10,7 @@
   "properties": {
     "values": {
       "description": "Default values for variables when computing readiness. Any values passed into the readiness check call will override these.",
-      "type": ["null", "object"],
+      "type": ["object", "null"],
```

Comment:
These changes to the readiness schema are unrelated -- I just noticed missing default code-completions on some of the field values while testing things out.
In this case here, having `null` first means you won't get the default `{}` code completion from the vscode json editor (it'll just have `null` as an option); flipping it makes it give you both.